recipes/cmdstan/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "cmdstan" %}
+{% set version = "2.27.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/stan-dev/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ff71f4d255cf26c2d8366a8173402656a659399468871305056aa3d56329c1d5
+
+build:
+  number: 0
+  binary_relocation: False
+
+requirements:
+  build:  
+    - {{ compiler('cxx') }}  # [not win]
+    - make                   # [not win]
+    - tbb-devel =2020.2      # [not win]
```

Comment:
```suggestion
```

Reply:
TBB has been an issue for us, so we ended up doing a sort of two-pronged approach: On Unix and MacOS, we depend on the conda-shipped TBB to make compilation simpler. On Windows, due to our dependence on `mingw64`'s `gcc`, we have to compile our own tbb, hence this line. 

We tried more or less every other combination in the above commits, but sadly this is what we found was required. The heavy use of templating in Stan's c++ code does not work under MSVC, so we need to take this somewhat odd route on Windows.

Reply:
Oh, I think you just meant that this library is in both `build` and `host`. My bad, I believe that is unintentional. The above comment can just serve as historical record, then.